x/group/spec/policies.md

```diff
@@ -0,0 +1,15 @@
+# Decision Policies
+
+```go
+type ThresholdDecisionPolicy struct {
+	// Specifies the number of votes that must be accumulated in order for a decision to be made by the group.
+	// A member gets as many votes as is indicated by their Weight field.
+	// A big integer is used here to avoid any potential vulnerabilities from overflow errors
+	// where large weight and threshold values are used.
+	DecisionThreshold sdk.Int `json:"decision_threshold"`
+}
+
+type PercentageDecisionPolicy struct {
+	Percent sdk.Dec `json:"percent"`
```

Comment:
Percentage may cause rounding issues. We have fractions in weave that solve this quite well.  https://github.com/iov-one/weave/blob/master/x/gov/codec.proto#L73

Reply:
This is a big decimal type that avoids floating point rounding errors.